From d4bbff5ce5705f877be326f32631b466daec5aed Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Tue, 1 May 2018 21:59:23 +0200
Subject: [PATCH] Fix pidl manpage sections

.TH header should match file name (i.e 3pm and not 3 for Parse::Pidl::NDR).

Signed-off-by: Mathieu Parent <math.parent@gmail.com>
Reviewed-by: Andrew Bartlett <abartlet@samba.org>
Reviewed-by: Garming Sam <garming@catalyst.net.nz>
---
 pidl/wscript | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/pidl/wscript b/pidl/wscript
index f4ff902812c..45cb17cea10 100644
--- a/pidl/wscript
+++ b/pidl/wscript
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-import os, Logs
+import os, string, Logs
 from samba_utils import MODE_755
 
 # This function checks if a perl module is installed on the system.
@@ -67,7 +67,8 @@ def build(bld):
     bld.SET_BUILD_GROUP('final')
     if 'POD2MAN' in bld.env and bld.env['POD2MAN'] != '':
         for src, manpage in pidl_manpages.iteritems():
-            bld(rule='${POD2MAN} -c "Samba Documentation" ${SRC} ${TGT}',
+            section = string.rsplit(manpage, ".", 1)[1]
+            bld(rule='${POD2MAN} -c "Samba Documentation" -s %s ${SRC} ${TGT}' % section,
                 shell=True,
                 source=src,
                 install_path=os.path.dirname(bld.EXPAND_VARIABLES('${MANDIR}/'+manpage)),
-- 
2.17.0

